h/static/scripts/directives/annotation.coffee

```diff
@@ -26,14 +26,15 @@ validate = (value) ->
 #
 # `AnnotationController` provides an API for the annotation directive. It
 # manages the interaction between the domain and view models and uses the
-# {@link annotator annotator service} for persistence.
+# {@link annotationMapper service} for persistence.
```

Comment:
The format of ngDoc @link is `{@link thingToLinkTo name to link to}` so I think you mean `{@link annotationMapper AnnotationMapper service}` or similar here.